tests/cases/conformance/jsdoc/jsdocLinkTag2.ts

```diff
@@ -0,0 +1,15 @@
+// @noUnusedLocals: true
+// @checkJs: true
+// @allowJs: true
+// @target: esnext
+// @noEmit: true
+
+// @filename: /a.js
+export class A {}
+
+// @filename: /b.js
+import { A } from "./a";
+
+/** {@link A} */
```

Comment:
what happens with bad references? Are those errors now? We need some tests of that -- or there might already be some.

Reply:
@sandersn I've added this [test](https://github.com/microsoft/TypeScript/blob/c46a01d38bf5c73890785a2ce43d3b3debb7739d/tests/cases/conformance/jsdoc/jsdocLinkTag5.ts) with unresolved link. With unresolved links, errors will be [ignored](https://github.com/microsoft/TypeScript/pull/47822/files#diff-d9ab6589e714c71e657f601cf30ff51dfc607fc98419bf72e04f6b0fa92cc4b8R36960)